components/size_aware_image.jsx

```diff
@@ -309,30 +309,44 @@ export default class SizeAwareImage extends React.PureComponent {
                     </div>
                     <span
                         className={classNames('image-preview-utility-buttons-container', 'image-preview-utility-buttons-container--small-image', {
-                            'image-preview-utility-buttons-container--small-image-no-copy-button': !enablePublicLink || !this.isInternalImage,
+                            'image-preview-utility-buttons-container--small-image-no-copy-button': !enablePublicLink,
                         })}
+                        // since there is a max-width constraint on images, a max-left clause follows.
+                        style={this.state.imageWidth > MIN_IMAGE_SIZE ? (
+                            // cases with no copy link buttons has different offsets than with copy buttons
+                            !enablePublicLink ? {left : `min(${wideImageButtonsOffset + 8}px, calc(100% - 24px)`}
+                            :  {left : `min(${wideImageButtonsOffset}px, calc(100% - 32px)`}
+                         ) : {}}
                     >
-                        {(enablePublicLink || !this.isInternalImage) && copyLink}
+                        {(enablePublicLink) && copyLink}
                         {download}
                     </span>
                 </div>
             );
         }
 
+        let utilityButtonsWrapper = null;
+        // handling external small images
+        if (this.state.isSmallImage && !this.isInternalImage) {
+            utilityButtonsWrapper = <></>;
```

Comment:
this is essentially the same as passing in `null`

Please re-consider using a ternary and `const` instead